source/onboarding/team/jenkins.html.md.erb

```diff
@@ -7,6 +7,10 @@ weight: 1
 
 # <%= current_page.data.title %>
 
+<div class="govuk-inset-text">
```

Comment:
If I use the standard markdown quote `>` the text gets wrapped in a `<blockquote>` and a `<p>` the `p` tag adds quite a bit of margin that makes it a bit weird. So I just used the standard component from the design system which looks better imo

Reply:
Can we fix the styling so that markdown quote renders correctly? (not necessarily in this PR). People have prob used MD quotes elsewhere and this might look even weirder if it looks different to those

Reply:
Used here: https://hmcts.github.io/ways-of-working/new-component/gitops-flux.html

and here:
https://hmcts.github.io/ways-of-working/troubleshooting/#flux-gitops